src/Symfony/Component/Intl/DateFormatter/IntlDateFormatter.php

```diff
@@ -635,7 +635,7 @@ protected function getDefaultPattern()
         if (self::NONE !== $this->timetype) {
             $patternParts[] = $this->defaultTimeFormats[$this->timetype];
         }
-        $pattern = implode(' ', $patternParts);
+        $pattern = implode(', ', $patternParts);
```

Comment:
because this is how the native IntlDateFormatter behaves (as spotted by some tests on Form when intl is disabled)